query.js

```diff
@@ -429,7 +429,14 @@ function parseQuery(query) {
                     let item = searchMap[k](queryItem._id, true);
                     if (queryItem.negative === true && queryItem._id !== "entry") {
                         for (let key in item) {
-                            item[key] = { $not: item[key] };
+                            let value = item[key];
+                            if (Array.isArray(value)) { // in the case that an array of queries is given, the entire array is negated with the $nor operator
```

Comment:
@ryanctan perhaps you could be a bit more specific with this comment. When you say "an array of queries is given", it sounds like you're referring to the original query passed in to `parseQuery`. However, you're actually referring to `item`, which is a value in `searchMap`.

So maybe you could word it to something like "in the case an array is returned from searchMap"?